spec/views/casa_cases/show.html.erb_spec.rb

```diff
@@ -12,7 +12,7 @@
 
   context "when there is court date" do
     let!(:casa_case) { create(:casa_case, :with_upcoming_court_date, casa_org: organization, case_number: "111") }
-    let(:date) { casa_case.court_dates.map(&:date).first.to_date.strftime("%B %d, %Y") }
+    let(:date) { casa_case.court_dates.map(&:date).first.to_date.strftime("%B %-d, %Y") }
```

Comment:
This totally works! What do you think about explicitly setting the court_date in the test, so the user can better understand the expectation without having to go look up the factory? I would also suggest removing the `let` statements and defining everything within the spec itself so it is easier to follow:


```
context "when there is court date" do
  it "renders casa case with court dates" do
    casa_case = create(:casa_case, case_number: "111")
    court_date = create(:court_case, casa_case: casa_case, date: Date.new(2023,5,6))

    assign(:casa_case, casa_case)
    render

    expect(rendered).to match("111")
    expect(rendered).to match("May 6, 2023")
  end
end
```